GetandTake/Pages/Categories/Image.cshtml

```diff
@@ -0,0 +1,9 @@
+﻿@page
+@model GetandTake.Pages.Categories.ImageModel
+@if (Model.Category.ImagePath != null)
+{
+    string imagePath = $"https://localhost:7005/" + Model.Category.ImagePath;
```

Comment:
Not a good idea to hardcode your URL here: if you decide to change host, it will cause issues.
You can specify URL in app settings and take it from there